keylime/tpm/tpm_abstract.py

```diff
@@ -89,37 +90,37 @@ def __init__(self, need_hw_tpm: bool = True) -> None:
         self.global_tpmdata = None
         self.tpmrand_warned = False
         self.defaults = {}
-        self.defaults["hash"] = algorithms.Hash.SHA1
+        self.defaults["hash"] = cast(algorithms.Hash, algorithms.Hash.SHA1)
```

Comment:
Without the cast it thinks this is a string.

Reply:
`Hash` has been imported. You can drop the import, or drop the `algorithms.` part

In any case I wonder why it is the case. According to the doc[1] support of enum in mypy

[1] https://mypy.readthedocs.io/en/stable/literal_types.html#enums